Assets/Scripts/StringmoreCamera/ShakeCamera.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace StringmoreCamera
+{
+    /// <summary>
+    /// Adds a rumbling shake to the MainCamera
+    /// </summary>
+    public class ShakeCamera : MonoBehaviour
+    {   
+        //TODO: possibly convert class to be a "camera effector" with more methods to cause different camera effects (shakes,zooms in/out etc)
+        private Coroutine _activeShake;
+        private Vector2 _originalCameraPos;
+
+        private void Awake()
+        {
+            _originalCameraPos = transform.localPosition;
+        }
+        /// <summary>
+        /// Public method that any class can shake the camera.
+        /// </summary>
+        /// <param name = "shakeDuration">the length of time to shake the camera for</param>
+        /// <param name = "shakeIntensity">the level of intensity to shake the camera (higher values cause more aggressive/violent shaking)</param>
+        /// <param name = "xShake/yShake">biases the direction in which the camera should shake</param>
+        public void Shake(float shakeDuration, float shakeIntensity, bool xShake, bool yShake) 
+        {
+            if(_activeShake != null) 
+            {
+                transform.localPosition = _originalCameraPos;
+            }
```

Comment:
I don't think this does anything since the shakeRoutine would immediately set the position from the previous frame.

If you're trying to avoid stacking, likely want to do `StopCoroutine(_activeShake)` instead.